app/exporters/patients_exporter.rb

```diff
@@ -191,4 +192,14 @@ def latest_blood_sugar_type(patient_summary)
     patient_summary.latest_blood_sugar_type.presence &&
       BLOOD_SUGAR_TYPES[patient_summary.latest_blood_sugar_type]
   end
+
+  def status(patient_summary)
+    patient_status = if patient_summary.ltfu?
+      :ltfu
+    else
+      patient_summary.status
+    end
```

Comment:
This kind of tree makes me wonder if we should compute the status, including LTFU, when populating the view.

Is that too much business logic inside view generation? Does patient summary have too many touchpoints to confidently adjust the values we put inside the status?

Reply:
This _could_ sit in the view too. The information we'll have to put in the view would be:
- what's the `LTFU_PERIOD`
- the status field in the CSV can also be `Lost to follow-up`. If we want to split the column again to `status` and `ltfu-ness`, this might need a change.

wydt? This is isn't very expensive to calculate in ruby and the win there is we have some flexibility about these two things.

Reply:
Hmm, yeah. We may not know enough just yet. Let's leave it as is.